_posts/2023-10-06-hatsumi-taikai.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title:  "Hatsumi Taikai 2024"
```

Comment:
I thought we agreed the name will not have the current year?

Reply:
> I thought we agreed the name will not have the current year?
we did